kio-client-java-spring/src/test/java/org/zalando/stups/clients/kio/spring/RestTemplateKioOperationsTest.java

```diff
@@ -94,9 +92,10 @@ public void testGetRangeOfApplications() throws ParseException {
         final ZonedDateTime modifiedBefore = now().plusDays(1);
         final ZonedDateTime modifiedAfter = now().minusDays(2);
 
-        mockServer.expect(requestTo(
-                format("%s/apps?modified_before=%s&modified_after=%s", //
-                        BASE_URL, toIsoString(modifiedBefore), toIsoString(modifiedAfter)))) //
+        mockServer.expect(requestTo(format("%s/apps?modified_before=%s&modified_after=%s", //
+                BASE_URL,
+                toIsoString(modifiedBefore).replaceAll("\\+", "%2B"),  //
```

Comment:
I would not use this in production code, either. However for testing, I think it is fine. Another good option would imho be, to use hardcoded values for the date times. This makes it even more obvious how the final result should look like.
e.g.

``` java
final ZonedDateTime modifiedBefore = ZonedDateTime.parse("2016-07-19T00:00:00.000Z");
final ZonedDateTime modifiedAfter = ZonedDateTime.parse("2016-06-11T00:00:00.000+02:00");
mockServer.expect(requestTo(format("%s/apps?modified_before=%s&modified_after=%s", 
        BASE_URL,
        "2016-07-19T00:00:00.000Z",  
        "2016-06-11T00:00:00.000%2B02:00"))) 
        .andExpect(method(GET)) 
        .andRespond(withSuccess(resource("/getApplications"), APPLICATION_JSON));
```